script/Ciel/Day 4/04_01_CIEL04_1B.txt

```diff
@@ -209,33 +207,34 @@ if you have to leave school early, please let us know. We would have immediately
 [sha:4cb03188f33c1abd285c858bda3904fb489a0e8a]{
 -- Page 53, Offset 8306.
 -- 何に遠慮しているかは知りませんが、兄さんは遠野家の長男だと言ったでしょう。使えるものはなんでも使ってください。そうでなくても、兄さんは人より体が弱いんですから」
-I do not know why you refrain so much, but I have told before that you are the eldest son of the Tohno family. Please use whatever is at your disposal. As it is, your body is already more fragile than most people, Nii-san"
+I don't know why you're always restraining yourself, but as I've said; you're the eldest son of the Tohno family. Please use whatever is at your disposal. As it is, your body is already more frail than most people, Nii-san"
 }
 [sha:29402c2c417099fe46d6bad0c60ec830da2c02dd]{
 -- Page 54, Offset 8307.
 -- 「昨日は突然の雨でしたから気圧の変化もあったのでしょうけど、あれぐらいで貧血を起こすなんて話になりません。
 // Needs double check to be sure
-"Yesterday it suddenly rained, so there must have been a change in air pressure, but I can't believe that you would get anemia from that.
+// looks right?
+"It suddenly rained yesterday so there was probably a slight change of atmospheric pressure, but that shouldn't have been enough to trigger your anemia.
```

Comment:
The sudden rain yesterday might've caused a slight change in atmospheric pressure, but that shouldn't have been enough to trigger your anemia.